clients/cloud/java-springboot/README.adoc

```diff
@@ -0,0 +1,172 @@
+= Spring Boot with Java Example Confluent Cloud Client
+
+NOTE: Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) using the Spring Boot using Java API.
+
+== Prerequisites
```

Comment:
@gAmUssA unfortunately the section Prerequisites format has changed since this PR was opened.  Can you please make it similar to https://github.com/confluentinc/examples/blob/5.4.1-post/clients/cloud/groovy/README.adoc ?

Reply:
updated